### ISSUE TITLE
feat(details): Add pictures in details panel and hovering

### ISSUE THREAD

**Issue by jolevesq** [fgpv-vpgf/fgpv-vpgf#1864]
Have the ability to display not only the link but the picture itself from the details panel and the hovering. From the filters table, display the picture in a lightbox when link is clicked.

**Comment by AleksueiR** [comment]
Do you mean render an url from the key:value pair in the details panel as image? Something lie this?
![](https://i.imgur.com/pdPtbTC.png)

How could we tell the url is an image, and not a simple link? Image urls won't necessarily end with `.png/jpg/gif/etc.`.

**Comment by jolevesq** [comment]
Yes like this. In GCViz we have a field type for this so it is simple. The way RCS register layers is there a way to specify a data field type?

We can say we only support images when url finish by ```png/jpg/gif/etc...``` so it will be not so complicated.

**Comment by AleksueiR** [comment]
Works for me.

**Comment by mweech** [comment]
I think there are two separate requirements here:

- under autolegend scenarios (i.e. catalogue viewer): we would support this if the link ends in an image type
- under thematic maps: allow specification of the fields that should be rendered as links vs images
